backend/core/Extensions/ObjectExtensions.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Linq;
+using System.Reflection;
+
+namespace Pims.Core.Extensions
+{
+    /// <summary>
+    /// ObjectExtensions static class, provides extension methods for generic objects.
+    /// </summary>
+    public static class ObjectExtensions
+    {
+        /// <summary>
+        /// Copies the public properties from the 'source' object to the 'destination' object.
+        /// It will only copy public properties that have a 'set' and are of the following types, primitive, enum, string.
+        /// Or an array of those same types.
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="destination"></param>
+        /// <typeparam name="ST"></typeparam>
+        /// <typeparam name="DT"></typeparam>
+        public static DT CopyValues<ST, DT>(this ST source, DT destination)
```

Comment:
Quick and dirty clone one object into another.   I believe the mapper can do this too, but will look into it later.

Reply:
yeah, I think the mapper does this by default if IgnoreNonMapped is not specified.